src/main/java/com/google/cloud/tools/maven/CloudSdkMojo.java

```diff
@@ -38,31 +38,27 @@
   @Parameter(property = "cloudSdkPath", required = false)
   protected File cloudSdkPath;
 
-  protected CloudSdk.Builder cloudSdkBuilder;
-
-  private final ProcessOutputLineListener gcloudOutputListener = new ProcessOutputLineListener() {
+  protected final ProcessOutputLineListener gcloudOutputListener = new ProcessOutputLineListener() {
```

Comment:
This feels like something that should go in the constructor? But that seems like a preference thing.